src/commands/deploy/index.ts

```diff
@@ -1,72 +1,162 @@
-import { Command, Flags } from "@oclif/core";
-import { execSync } from "node:child_process";
+import { Command, Flags, CliUx } from "@oclif/core";
 import { readFileSync, writeFileSync } from "node:fs";
 import path = require("node:path");
+import { CodePromise, Abi } from "@polkadot/api-contract";
+import { Keyring } from "@polkadot/keyring";
+import { readJSONSync } from "fs-extra";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
+import { ChainApi } from "../../lib/substrate-api";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { Listr } from "listr2";
+import { ensureSwankyProject } from "../../lib/command-utils";
 export class DeployContract extends Command {
   static description = "Deploy contract to a running node";
 
   static flags = {
+    account: Flags.string({
+      required: true,
+      options: ["Alice", "Bob", "Charlie", "Dave", "Eve", "Ferdie"],
```

Comment:
I think this should be handled by the keyring object instead of hard-coding the account names. For example, the user can pass the string name as an argument and the script will search for the matching account in the keyring (as an example)

Reply:
I deliberately didn't touch the user and key management, will handle that module as a separate PR, and update here accordingly.

Reply:
Understood